grakn-graph/src/main/java/ai/grakn/graph/internal/ValidateGlobalRules.java

```diff
@@ -186,7 +185,7 @@ private ValidateGlobalRules() {
      * @return An error message if the conceptType  abstract and has incoming isa edges
      */
     static Optional<String> validateIsAbstractHasNoIncomingIsaEdges(TypeImpl conceptType){
-        if(conceptType.isAbstract() && conceptType.getVertex().edges(Direction.IN, Schema.EdgeLabel.ISA.getLabel()).hasNext()){
+        if(conceptType.isAbstract() && ((TypeImpl) conceptType.currentShard()).getIncomingNeighbours(Schema.EdgeLabel.ISA).findAny().isPresent())  {
```

Comment:
This will accept an abstract type with multiple shards, even if one of those shards has instances.